.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
 - npm install
 - bower install
 after_success:
-- sed -i -- 's/SF:ember-frost-sidebar\/\(.*\)/SF:addon\/\1.js/' coverage/lcov.info && rm -f coverage/lcov.info--
```

Comment:
Ok. I talked to David about and as per his suggestion is to simply copy good .travis file and update/overwrite the content such as secure key that is specific to my project.